packages/mui/test/GridSnap.test.tsx

```diff
@@ -0,0 +1,278 @@
+import { gridTests } from '@rjsf/snapshot-tests';
+
+import Form from '../src';
+
+// The `TextareaAutosize` code reads the following data from the `getComputedStyle()` function in a useEffect hook
+jest.spyOn(window, 'getComputedStyle').mockImplementation(() => {
+  return {
+    width: 100,
+    'box-sizing': 10,
+    'padding-bottom': 1,
+    'padding-top': 1,
+    'border-bottom-width': 1,
+    'border-top-width': 1,
+  } as unknown as CSSStyleDeclaration;
+});
```

Comment:
is this mock implementation copied from another test suite? would it make sense to put it all in one place?

Reply:
Refactored